imodel-schema-validator/package.json

```diff
@@ -67,9 +66,10 @@
     ],
     "extends": "plugin:@itwin/itwinjs-recommended",
     "rules": {
+      "@typescript-eslint/dot-notation": "off",
       "prefer-template": "off",
       "dot-notation": "off",
```

Comment:
Can this be removed now that we're using typescript-eslint?

Reply:
Done